### PR TITLE
TIP-860: Move front resources

### PR DESCRIPTION
**Pull request by arnolanglade**
**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added legacy Behats               | -
| Added acceptance tests            | -
| Added integration tests           | -
| Changelog updated                 | -
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: src/Akeneo/Pim/Enrichment/Bundle/Resources/config/providers.yml [hunk 1]
```diff
@@ -30,3 +33,30 @@ services:
...
+        class: '%pim_enrich.provider.structure_version.class%'
+        deprecated: 'The "%service_id%" service is deprecated. Use "pim_structure_version.provider.structure_version.product" instead'
+        arguments:
+            - '@doctrine'
+        calls:
+            - [addResource, ['%pim_catalog.entity.association_type.class%']]
+            - [addResource, ['%pim_catalog.entity.attribute.class%']]
+            - [addResource, ['%pim_catalog.entity.attribute_group.class%']]
+            - [addResource, ['%pim_catalog.entity.category.class%']]
+            - [addResource, ['%pim_catalog.entity.channel.class%']]
```
Comment (jjanvier) [inline src/Akeneo/Pim/Enrichment/Bundle/Resources/config/providers.yml]: shouldn't we separate channel, family, attribute, attribute group and locale?

File: src/Akeneo/Pim/Enrichment/Bundle/Resources/config/providers.yml [hunk 2]
```diff
@@ -30,3 +33,30 @@ services:
...
+            - [addResource, ['%pim_catalog.entity.group.class%']]
+            - [addResource, ['%pim_catalog.entity.locale.class%']]
+
+    pim_enrich.provider.structure_version.group:
+        class: '%pim_enrich.provider.structure_version.class%'
+        deprecated: 'The "%service_id%" service is deprecated. Use "pim_structure_version.provider.structure_version.group" instead'
+        arguments:
+            - '@doctrine'
+        calls:
+            - [addResource, ['%pim_catalog.entity.attribute.class%']]
```
Comment (jjanvier) [inline src/Akeneo/Pim/Enrichment/Bundle/Resources/config/providers.yml]: shouldn't we separate this service and move it elsewhere?

File: src/Akeneo/Pim/Enrichment/Bundle/Resources/config/providers.yml (hunk 1 above)
Comment (anaelChardan) [reply src/Akeneo/Pim/Enrichment/Bundle/Resources/config/providers.yml]: The problem is that the service isn't tagged so to know the impact it's very difficult but yes, ideally you're true

Comment (arnolanglade) [reply src/Akeneo/Pim/Enrichment/Bundle/Resources/config/providers.yml]: Providers are an old way to manage front stuff, we can let them here.

File: src/Akeneo/Pim/Structure/Bundle/Resources/translations/jsmessages.en_US.yml [hunk 3]
```diff
@@ -0,0 +1,268 @@
+pim_enrich.entity.channel:
```
Comment (anaelChardan) [inline src/Akeneo/Pim/Structure/Bundle/Resources/translations/jsmessages.en_US.yml]: not in Channel ?

Comment (jjanvier) [reply src/Akeneo/Pim/Structure/Bundle/Resources/translations/jsmessages.en_US.yml]: oups

**Comment by jjanvier** [comment]
7 behats failing
5 are OK locally
2 failing on master

merging...

**Pull request merged**